src/main/resources/format-pages.txt

```diff
@@ -604,9 +604,11 @@ opennessRating = Very good
 presenceRating = Good
 utilityRating = Fair
 reader = DicomReader
+writer = DicomWriter
 mif = true
 notes = * DICOM stands for "Digital Imaging and Communication in Medicine". \n
 * Bio-Formats supports both compressed and uncompressed DICOM files. \n
+* Support for reading and writing DICOM whole slide images (DICOM WSI format) was implemented through collaboration with `NCI Imaging Data Commons <https://imaging.datacommons.cancer.gov/>`_  and has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Task Order No. HHSN26110071 under Contract No. HHSN2612015000031. \n
```

Comment:
As shown by https://merge-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-linkcheck/683/ and

```
$ curl -IL https://imaging.datacommons.cancer.gov/
HTTP/1.1 500 Internal Server Error
Date: Thu, 09 Dec 2021 09:12:52 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 12229
Server: gunicorn/19.6.0
Vary: Cookie
X-Frame-Options: SAMEORIGIN
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Set-Cookie: csrftoken=NFcBeGjVhSYOVqy2RrhnvTVTamITXA5RSZl7GPTCRqf6sOOSRjqF9V7GCmR8hJSw; expires=Thu, 08 Dec 2022 09:12:52 GMT; Max-Age=31449600; Path=/; SameSite=Lax; Secure
Via: 1.1 google
Alt-Svc: clear
 ```

Should we add to the linkcheck ignore list for now and report upstream?

Reply:
Possible less direct, I would propose https://datacommons.cancer.gov/repository/imaging-data-commons  as an alternative link here that would return 200